crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1333,6 +1339,37 @@ impl Default for MainThreadValidator {
     }
 }
 
+pub trait ManyRes<'w> {
```

Comment:
This needs doc strings. It should link back to `get_many_res`.

Ditto for `ManyResMut`.